### PR TITLE
Migrates infodocs, CSP, message_duplicates and navigation e2e tests

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

https://github.com/medic/angular10-migration/issues/84
https://github.com/medic/angular10-migration/issues/72
https://github.com/medic/angular10-migration/issues/81
https://github.com/medic/angular10-migration/issues/141

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @newtewt 
Would you, please, have a look? 
Thank you!

File: tests/page-objects/common/common.po.js [hunk 1]
```diff
@@ -89,7 +92,7 @@ module.exports = {
     await helper.waitElementToPresentNative(element(by.css('.sync-status .success')));
   },
 
-  checkUserSettings: () => {
+  checkUserSettings: async () => {
```
Comment (newtewt) [inline tests/page-objects/common/common.po.js:95]: Seems to be no `await` here. Is it needed to be async?

**Review by newtewt — APPROVED** [review]
Just one question, Nice moving these over!

File: tests/page-objects/common/common.po.js (hunk 1 above)
Comment (dianabarsan) [reply tests/page-objects/common/common.po.js:95]: Great eye! Thanks for catching this! I've added the necessary awaits.

**Pull request merged**